pom.xml

```diff
@@ -109,6 +109,7 @@
 
     <!-- test library versions -->
     <junit.version>4.12</junit.version>
+    <hamcrest-all.version>1.3</hamcrest-all.version>
```

Comment:
`jUnit` depends on `hamcrest-core 1.3` and we already used it in the project. I placed it right after jUnit.